adapters/silverpush/devicetype.go

```diff
@@ -0,0 +1,44 @@
+package silverpush
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+)
+
+func isMobile(ua string) bool {
+	isValidMobile := regexp.MustCompile(`(ios|ipod|ipad|iphone|android)`)
+	return isValidMobile.MatchString(strings.ToLower(ua))
+}
+
+func isCTV(ua string) bool {
+	isCtv := regexp.MustCompile(`(smart[-]?tv|hbbtv|appletv|googletv|hdmi|netcast\.tv|viera|nettv|roku|\bdtv\b|sonydtv|inettvbrowser|\btv\b)`)
+	return isCtv.MatchString(strings.ToLower(ua))
+}
+
+// Check valid Eids
+func IsValidEids(eids []openrtb2.EID) bool {
```

Comment:
Please start this name with a lower case letter. There is no need to export this function outside of your adapter package.

Reply:
Fixed
[commit](https://github.com/prebid/prebid-server/pull/2957/commits/cc30407f798a60b4550738f3802ffe7a19f81493)